configuration/lambda_config.json

```diff
@@ -27,10 +27,10 @@
     },
     "loggers": {
       "origin-response": {
-        "level": "WARNING"
+        "level": "$ORIGIN_RESPONSE_LOGGER_LEVEL"
```

Comment:
You should have some default values for these vars.  If you don't, then this new version of the code can't be deployed by the current version of the playbooks & pipeline.

I'd suggest doing it like this in `scripts/build-package` :

```
export ORIGIN_RESPONSE_LOGGER_LEVEL=${ORIGIN_RESPONSE_LOGGER_LEVEL:-WARNING}
export ORIGIN_REQUEST_LOGGER_LEVEL=${ORIGIN_REQUEST_LOGGER_LEVEL:-WARNING}
```

That will set the env vars to WARNING if and only if they are not already set.  Must be done before the call to 'envsubst'.